addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -752,6 +752,12 @@ MockServer.include({
             });
             if (channel.channel_type === 'channel') {
                 delete res.members;
+            } else {
+                res['seen_partners_info'] = [{
+                    partner_id: this.currentPartnerId,
+                    seen_message_id: lastMessageId,
+                    fetched_message_id: lastMessageId,
+                }]
```

Comment:
Isn't it a bit too much of a simplification to set the last message of the thread always seen/fetched for the current partner?

Ideally it should at least come from a key from mock model so we can control it with `this.data`.

Also missing `;` at the end here.

Reply:
The best way to do it is to implement channel partner in mock server but it's not a simple change
I'm not that confortable to add a fake mock model field but maybe it's better ...
Or maybe we could use channel.seen_message_id ?

Reply:
Yes, seen_message_id in channel is the mock server fake field for the seen feature of the current partner.